doc/guides/connection_management.rst

```diff
@@ -189,38 +161,57 @@ Example:
 ::
 
     n, m, p = 10, 12, 0.2
-    A = Create("iaf_psc_alpha", n)
-    B = Create("iaf_psc_alpha", m)
+    A = nest.Create('iaf_psc_alpha', n)
+    B = nest.Create('iaf_psc_alpha', m)
     conn_dict = {'rule': 'pairwise_bernoulli', 'p': p}
-    Connect(A, B, conn_dict)
+    nest.Connect(A, B, conn_dict)
+
+symmetric-pairwise-bernoulli
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For each possible pair of nodes from ``pre`` and ``post``, a connection
+is created with probability ``p`` from ``pre`` to ``post``, as well as
+a connection from ``post`` to ``pre`` (two connections in total). To
+use the 'symmetric_pairwise_bernoulli' rule, ``allow_autapses`` must
+be False, and ``make_symmetric`` must be True.
+
+Example:
+
+::
+
+    n, m, p = 10, 12, 0.2 
+    A = nest.Create('iaf_psc_alpha', n) 
+    B = nest.Create('iaf_psc_alpha', m) 
+    conn_dict = {'rule': 'symmetric_pairwise_bernoulli', 'p': p, 'allow_autapses': False, 'make_symmetric': True} 
+    nest.Connect(A, B, conn_dict)
 
 .. _synapse_spec:
 
 Synapse Specification
--------------------------
+---------------------
 
-The synapse properties can be given as a string or a dictionary. The
-string can be the name of a pre-defined synapse which can be found in
-the synapsedict (see  :ref:`synapse-types`) or a manually defined
-synapse via `CopyModel()`.
+The synapse properties can be given as a string, a ``CollocatedSynapse``
+object, or a dictionary. The string can be the name of a pre-defined
+synapse which can be found in the synapsedict (see  :ref:`synapse-types`)
+or a manually defined synapse via `CopyModel()`.
 
 Example:
 
 ::
 
     n = 10
-    A = Create("iaf_psc_alpha", n)
-    B = Create("iaf_psc_alpha", n)
-    CopyModel("static_synapse","excitatory",{"weight":2.5, "delay":0.5})
-    Connect(A, B, syn_spec="excitatory")
+    A = nest.Create('iaf_psc_alpha', n)
+    B = nest.Create('iaf_psc_alpha', n)
+    nest.CopyModel('static_synapse','excitatory',{'weight': 2.5, 'delay': 0.5})
+    nest.Connect(A, B, syn_spec='excitatory')
 
 Specifying the synapse properties in a dictionary allows for distributed
-synaptic parameter. In addition to the key ``model`` the dictionary can
+synaptic parameters. In addition to the key ``synapse_model`` the dictionary can
 contain specifications for ``weight``, ``delay``, ``receptor_type`` and
 parameters specific to the chosen synapse model. The specification of
 all parameters is optional. Unspecified parameters will use the default
 values determined by the current synapse model. All parameters can be
-scalars, arrays or distributions (specified as dictionaries). One
+scalars, arrays or distributions (specified as a ``nest.Parameter``). One
```

Comment:
Perhaps we could have parameter subsections titled "scalar", "array" and "distributed". What do you think?

Reply:
Will do!